tests/cpp/test_move_pad.cpp

```diff
@@ -23,9 +23,8 @@ using testing::UnorderedElementsAre;
 
 class MovePadTest : public NVFuserTest {
  protected:
-  void SetUp() override {
+  MovePadTest() {
```

Comment:
https://google.github.io/googletest/faq.html#CtorVsSetUp

Reply:
The FAQ says that the test constructor is preferred besides for a few exceptions, which one applies in this case?

Reply:
None. So constructor

Reply:
Haha, thanks. I read it backwards.